shapeworks_cloud/core/rest.py

```diff
@@ -18,7 +18,7 @@
 from . import filters, models, serializers
 from .tasks import groom, optimize
 
-DB_WRITE_ACCESS_LOG_FILE = Path(os.path.expanduser('~'), 'logging', 'db_write_access.log')
+DB_WRITE_ACCESS_LOG_FILE = Path('/tmp', 'logging', 'db_write_access.log')
```

Comment:
This should be `tempfile.gettempdir()`, not `/tmp`

Reply:
Gotcha. Changed in [24c812d](https://github.com/girder/shapeworks-cloud/pull/328/commits/24c812dfde2c43c70a8f9023075b74bc848db69a)